build.gradle

```diff
@@ -167,26 +169,30 @@ allprojects {
 }
 
 // shell scripts, provided they have ".sh" extension
-tasks.register('shellCheck', Exec) {
-	description 'Check all shell scripts using shellcheck, if available'
-	group 'lint'
-
-	inputs.files fileTree('.').exclude('**/build').include('**/*.sh')
-	outputs.file "$temporaryDir/log"
-
-	doFirst {
-		// quietly succeed if "shellcheck" is not available
-		executable 'shellcheck'
-		final def execPaths = System.getenv('PATH').split(File.pathSeparator)
-		final def isAvailable = execPaths.any { file("$it/$executable").exists() }
-		if (!isAvailable) executable 'true'
-
-		args inputs.files
-
-		final def consoleOutput = System.out
-		final def fileOutput = new FileOutputStream(outputs.files.singleFile)
-		final def bothOutput = new org.apache.tools.ant.util.TeeOutputStream(consoleOutput, fileOutput)
-		standardOutput = errorOutput = bothOutput
+if (System.getProperty('os.name').startsWith('Windows')) {
+	tasks.register('shellCheck')
```

Comment:
Why does it need to be registered here?  Maybe add a comment?

Reply:
Clarified in 6a72360f106e1ca7b408749af2753ff718f19ed1.